src/include/sof/lib/uuid.h

```diff
@@ -55,6 +57,14 @@ struct sof_uuid_entry {
 	const char name[UUID_NAME_MAX_LEN];
 };
 
+/** \brief Check if the provided UUID is a Nil one. */
```

Comment:
@keyonjie sorry, I still dont understand why we would have extended data with a NULL UUID. UUID is the first user of the extended data.

Reply:
Not needed